### ISSUE TITLE
code view: also show image files touched by the user

### ISSUE THREAD

**Issue by damithc** [reposense/RepoSense#406]
Suggestion: it is good if we can also show image files that can be attributed to the author e.g., diagrams added

**Comment by eugenepeh** [comment]
Hmm one concern about this is that, the users can't annotate the image to claim it as theirs. Hence, the easiest approach would be to credit the image to the last modifier. However, this can cloud the judgement of the reviewer to believe that the entire image was done by the author.

Another approach that I can think of is to create an "image section" under the repository. This image section will purely show the list of images and the corresponding commits that have made changes to it. This would be somewhat similar to the commits view that we were working on previously.

**Comment by damithc** [comment]
yes, for images we can also list previous authors who modified the file.
